fhir-bucket/src/main/java/com/ibm/fhir/bucket/persistence/MergeResourceTypes.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2020, 2021
```

Comment:
Are we losing this space intentionally?
I've been using the space pretty consistency, so I think it will lead to lots of churn.

Reply:
ack, not intentional, I flipped it accidently